src/SDKs/Automation/Automation.Tests/Customizations/AutomationManagementClient.Customization.cs

```diff
@@ -3,12 +3,11 @@
 
 using System;
 using System.Collections.Generic;
-using System.Net.Http;
 using Microsoft.WindowsAzure.Management.Automation;
 
-namespace Microsoft.WindowsAzure.Management.Automation
+namespace Automation.Tests.Customizations
 {
-    public partial class AutomationManagementClient
+    public class AutomationManagementClient
     {
         public static List<T> ContinuationTokenHandler<T>(Func<string, ResponseWithSkipToken<T>> listFunc)
         {
```

Comment:
Do we need this? I do not think we have an SkipTokens
only skip and take

Reply:
I will remove the Customizations folder. We do not need this now.